libs/entitlement/qa/entitlement-test-plugin/build.gradle

```diff
@@ -23,6 +23,7 @@ esplugin {
 dependencies {
   compileOnly project(':server')
   compileOnly project(':libs:logging')
+  compileOnly project(":libs:entitlement")
```

Comment:
Not blocking, but I wonder why you had to add this?

Reply:
Oh I don't need it anymore. I can remove it.

In an earlier draft, I had one of the tests catch `NotEntitledException` to make sure that the right part of the test was throwing. That turned out not to be necessary in the end.